src/components/index.js

```diff
@@ -25,6 +25,7 @@ export { default as DatePicker } from './DatePicker';
 export { default as DateTimePicker } from './DateTimePicker';
 export { default as GMap } from './GMap';
 export { default as GoogleAddressLookup } from './GoogleAddressLookup';
+export { default as Icon } from './Icon';
```

Comment:
For now, we will not have an Icon component, maybe we can have a further discussion about whether it is needed or useful.